docker/Dockerfile

```diff
@@ -1,15 +1,13 @@
 # python 3
 FROM continuumio/miniconda3
 
-MAINTAINER Attawit Kittikrairit
```

Comment:
put me as the maintainer instead

`MAINTAINER K. Shankari (shankari@eecs.berkeley.edu)`